src/helpers/Robots/Robots.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Sitemap helper.
+ * @module helpers/Sitemap
+ */
+
+import superagent from 'superagent';
+import cookie from 'react-cookie';
+import { toPublicURL } from '@plone/volto/helpers';
+import { settings } from '~/config';
+
+/**
+ * Generate robots. Get robots from plone
+ * @function generateRobots
+ * @param {Object} req Request object
+ * @return {string} Generated robots
+ */
+export const generateRobots = (req) =>
+  new Promise((resolve) => {
+    //const url = `${req.protocol}://${req.get('Host')}`;
+    const request = superagent.get(`${settings.apiPath}/robots.txt`);
```

Comment:
should also use settings.internalApiPath

Reply:
in production we use only RAZZLE_API_PATH. Isn't it correct?

We could change to 
``${settings.internalApiPath ?? settings.apiPath}/robots.txt``

what do you think?

Reply:
Yes, correct. superagent calls should use the internalApiPath, if set, as much as possible.

Reply:
correct!

> Yes, correct. superagent calls should use the internalApiPath, if set, as much as possible.

fixed